### ISSUE TITLE
Error with CRA 1.0.7

### ISSUE THREAD

**Issue by stereobooster** [styleguidist/react-styleguidist#458]
When something isn’t working, include:

- `styleguide.config.js` none.
- webpack "2.6.1"
- MacOS
- Reproducible demo: 

```
git clone https://github.com/stereobooster/react-lazy-load.git
git checkout my-fork
cd examples/basic
sh ./link.sh
yarn
yarn run styleguide
```

```
yarn run styleguide
yarn run v0.24.6
$ styleguidist server
Loading webpack config from:
react-scripts/config/webpack.config.dev

/react-lazy-load/examples/basic/node_modules/react-styleguidist/bin/styleguidist.js:119
		throw err;
		^

Error: The NODE_ENV environment variable is required but was not specified.
    at Object.<anonymous> (react-lazy-load/examples/basic/node_modules/react-scripts/config/env.js:22:9)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (react-lazy-load/examples/basic/node_modules/react-scripts/config/webpack.config.dev.js:22:30)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
error Command failed with exit code 1.
```

**Comment by sapegin** [comment]
Thanks for the repro! I’ll make a new release as soon as I figure out why CI is failing ;-|

**Comment by sapegin** [comment]
Should be fixed in [5.3.0](https://github.com/styleguidist/react-styleguidist/releases/tag/v5.3.0).
